docker-compose.test.yml

```diff
@@ -115,7 +116,8 @@ services:
         KERAS_PACKAGE: git+https://github.com/keras-team/keras.git
         PYTORCH_PACKAGE: torch-nightly
         PYSPARK_PACKAGE: pyspark==2.4.0
-  test-cpu-mpich-py2_7-tf1_12_0-keras2_2_2-torch1_0_0-mxnet1_4_0-pyspark2_4_0:
+        MXNET_PACKAGE: mxnet --pre
```

Comment:
why not use the same 1.5.0b20190412 as above here?

Reply:
See above answer.